# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: media/libcubeb/moz.yaml
# Commit: 3ba614682b62
# Full Hash: 3ba614682b62df868e89da1d7705b3191954f0fc
# Author: Alex Chronopoulos <achronop@gmail.com>
# Date: 2019-04-25 16:18:07
# Description:
#   Bug 1545279 - Update cubeb from upstream to 3570749. r=padenot
#   
#   Pick commits:
#   3570749 - wasapi: tie monitor lifetime with notification client lifetime. BMO 1545279 (#505)
#   162625a - test: add option to get the posotion of a stream (#504)
# ==============================================================================

diff -r e1ab2cda0424 -r 3ba614682b62 media/libcubeb/moz.yaml
--- a/media/libcubeb/moz.yaml	Wed Apr 24 14:52:13 2019 +0000
+++ b/media/libcubeb/moz.yaml	Thu Apr 25 09:22:35 2019 +0000
@@ -19,5 +19,5 @@
   license: "ISC"
 
   # update.sh will update this value
-  release: "241e3c7b8a6ce76ad9e075ee5761cd4d0906bc16 (2019-04-16 17:39:01 +0200)"
+  release: "3570749942a4fea2b7e372833cb8374ce42cf389 (2019-04-24 13:04:01 +1200)"
 